packages/smooth_app/lib/helpers/temp_product_list_share_helper.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:openfoodfacts/openfoodfacts.dart';
+
+// TODO(m123): Move this to off-dart
+Uri shareProductList(List<String> barcodes) {
+  final StringBuffer buffer = StringBuffer();
```

Comment:
why not something like

final String commaSeparatedBarcodes = barcodes.join(', ');

That's simpler and would get rid of the extra comma at the end.